src/commands/coin/coinInfo.ts

```diff
@@ -0,0 +1,46 @@
+import { Message, MessageEmbed } from 'discord.js';
+import { CommandoClient, CommandoMessage } from 'discord.js-commando';
+import { BaseCommand } from '../../utils/commands';
+import { EMBED_COLOUR } from '../../utils/embeds';
+
+class CoinInfo extends BaseCommand {
+  constructor(client: CommandoClient) {
+    super(client, {
+      name: 'coin-info',
+      aliases: ['ci', 'coininfo', 'codeycoin info'],
+      group: 'coin',
+      memberName: 'coin-info',
+      description: 'Info about Codey Coin, including ways to get and how to spend Codey Coins.',
+      examples: [`${client.commandPrefix}coin-info`, `${client.commandPrefix}ci`, `${client.commandPrefix}coininfo`]
+    });
+  }
+  infoEmbed = new MessageEmbed()
+    .setColor(EMBED_COLOUR)
+    .setTitle('🪙 About Codey Coin 🪙')
+    .setThumbnail('') // Thumbnail, if needed?
```

Comment:
is the thumbnail the giant coin emote in the upper right of the embed? if so we can keep that

Reply:
Ok, sounds good. I was worried it was too big, I'll add it back in